pilbox/app.py

```diff
@@ -138,7 +138,7 @@ def __init__(self, **kwargs):
         tornado.web.Application.__init__(self, self.get_handlers(), **settings)
 
     def get_handlers(self):
-        return [(r"/", ImageHandler)]
+        return [(r"/(.*)", ImageHandler)]
```

Comment:
I'm not sure what you're confused about. I had to make these changes to get the code to work. I couldn't just change this one line. Happy to discuss on slack if you want me to go over the changes